lib/openh264.py

```diff
@@ -78,3 +79,10 @@ def ResultData(self, encodedfile):
     more_results = {}
     more_results['frame'] = file_codec.FfmpegFrameInfo(encodedfile)
     return more_results
+
+  def EncoderVersion(self):
+    # openh264 doesn't appear to have a built-in version string. Use the
+    # git checksum instead.
+    git_hash = subprocess.check_output(
+        'cd third_party/openh264; git log --format="%h %ad" -1', shell=True)
```

Comment:
It won't - that's why I had to write this as a "shell=True" subprocess rather than "shell=False" (which is usually preferable).